fcs/models.py

```diff
@@ -89,11 +89,27 @@ def __unicode__(self):
         return self.name
 
 
+# Many-to-many table connecting Undertakings and BusinessProfiles
+undertaking_businessprofile = db.Table(
+    'undertaking_businessprofile',
+    db.Column(
+        'undertaking_id', db.Integer, db.ForeignKey('undertaking.id')),
+    db.Column(
+        'businessprofile_id', db.Integer, db.ForeignKey('businessprofile.id')),
+)
+
+
 class BusinessProfile(SerializableModel, db.Model):
     __tablename__ = 'businessprofile'
 
     id = Column(Integer, primary_key=True)
     highleveluses = Column(String(255))
+    domain = Column(String(32), default="FGAS")  # TODO: Use ChoiceType
+    undertakings = relationship(
```

Comment:
I would keep this relation in the Undertaking class as this is the type that is used the most in the application.

Reply:
As far as I understand, the `backref` parameter sent to the relationship definition allows us to get related objects both from Undertaking and BusinessProfile instances. For example:

```
> undertaking.businessprofiles
[<BusinessProfile1>, <BusinessProfile2>, ...]
> businessprofile.undertakings
[<Undertaking1>, <Undertaking2>, ...]
```

Reply:
Yes, that's true. I said that I would keep that relation in the Undertaking because I thought it would be easier to follow its relationships with the other classes if it was there( as you can see this class has a lot of fields) . Anyway, great job with this task :)